value/unstructured.go

```diff
@@ -53,6 +54,21 @@ func FromYAML(input []byte) (Value, error) {
 	return v, nil
 }
 
+// FromJSON is a helper function for reading a JSON document
+func FromJSON(input []byte) (Value, error) {
+	var decoded interface{}
+
+	if err := yaml.Unmarshal(input, &decoded); err != nil {
```

Comment:
I understand that json is a subset of yaml, but why is it using the yaml library here in the FromJSON function?